docs/references/javascript/organization/invitations.mdx

```diff
@@ -7,7 +7,32 @@ description: These are all methods on the Organization class that allow you to m
 
 These are all methods on [the Organization class](/docs/references/javascript/organization/organization) that allow you to manage the invitations to an organization.
 
-## `getPendingInvitations()`
+## `getInvitations()`
+
+Retrieves a list of invitations for the currently active organization.
+
+```typescript
+function getInvitations(params?: GetInvitationsParams): Promise<PaginatedResources<OrganizationInvitation>>;
+```
+
+### `GetInvitationsParams`
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `initialPage?` | `number` | A number that can be used to skip the first n-1 pages. For example, if `initialPage` is set to 10, it is will skip the first 9 pages and will fetch the 10th page. |
+| `pageSize?` | `number` | A number that indicates the maximum number of results that should be returned for a specific page. |
+| `status?` | `'pending' \| 'accepted' \| 'revoked'` | The status an invitation can have. |
+
+### Returns
+
+| Type | Description |
+| --- | --- |
+| <code>Promise\<[PaginatedResponse][pag-res-ref]\<[OrganizationInvitation][org-inv-ref]\>></code> | This method returns a `Promise` which resolves with a list of [`OrganizationInvitation`][org-inv-ref] objects. |
```

Comment:
Oh it looks like this page is using [orginv-ref] instead of [org-inv-ref]

Also, don't forget to add this reference to the bottom of the page:

```
[pag-res-ref]: /docs/references/javascript/types/paginated-response
```